src/Westeros/SouthOfTheWall/Lexer.x

```diff
@@ -33,10 +36,20 @@ tokens :-
 <string>    $printable                              { pushToString }
 <string>    .                                       { invalidCharacter }
 
+-- Literals
+<0>     $digits+$white+golden$white+coins                           ;  
+<0>     $digits+.$digits$white+drops$white+of$white+poison          ; 
```

Comment:
The . need to be escaped. And we need to allow negative numbers. Put [-]{0,1} at the start of this regexes